packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.spec.js

```diff
@@ -32,6 +30,10 @@ describe('Tasks', function () {
     before(function () {
       const task = taskRegistry.get(taskType)
       TaskComponent = observer(task.TaskComponent)
+      // DrawingTask, TranscriptionTask, DataVisAnnotationTask, TextTask all use instruction
+      // SingleChoiceTask, MultipleChoiceTask use question
+      // keys that aren't defined on certain task models are ignored
+      // but missing keys that aren't an optional or maybe type will throw an error
       const taskSnapshot = {
```

Comment:
Maybe I should have named this `genericTaskSnapshot` since it's a snapshot that can be used to create any task.